src/components/Home/components/NewProblemsForm/index.js

```diff
@@ -142,13 +149,31 @@ export default class NewProblemsForm extends Component {
                 </div>
             </div>
         var doneButton = this.props.editing ? null :
```

Comment:
It's confusing what's happening in this conditional statement. Could you refactor it?